xcodeproj/internal/top_level_targets.bzl

```
@@ -225,10 +226,14 @@ def process_top_level_target(
     additional_files.extend(list(modulemaps.files))
 
     app_icon_info = app_icons.get_info(ctx, automatic_target_info)
+    alternate_icons_info = alternate_icons.get_info(ctx, automatic_target_info)
+    if alternate_icons_info:
+        additional_files.extend(list(alternate_icons_info.alternate_icons_files))
```

```suggestion
        additional_files.extend(alternate_icons_info.alternate_icons_files)
```

the above `list()` is because `modulemaps.files` is a `tuple`.